src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-list/iscsi-target-list.component.html

```diff
@@ -14,6 +14,7 @@
   </ng-container>
 </cd-alert-panel>
 
+<legend i18n>Targets</legend>
```

Comment:
As this happens in the same way every table, what about moving this into the datatable HTML and adding a new input arg?

Reply:
Good idea @epuertat ! Will apply it